examples/oft-solana/Anchor.toml

```diff
@@ -0,0 +1,19 @@
+[toolchain]
+anchor_version = "0.29.0"
+
+[features]
+seeds = false
+skip-lint = false
+
+[programs.localnet]
+oft = "CUinP3FqxxPGxhnmTSMDJM9rnKW6urhDugecXV13eZdK"
```

Comment:
If I am not mistaken, don't we want folks to deploy their own program with their own account?

Reply:
Solana is weird and so you create keypairs ahead of time and then deploy to them.

You can run `anchor keys sync` to sync to your keypairs for your wallet.

Reply:
> Each Anchor project has it's own program address when it's first built. Since you've just copied my code, the program address at the top of your lib.rs won't match yours. To generate a keypair, run anchor build, and then run anchor keys sync to set it in your Anchor.toml and in the id field of your lib.rs.

https://solana.com/developers/guides/getstarted/full-stack-solana-development

Reply:
Yeah so this is one of the tricky parts when it comes to having this project in a CI/CD environment. It goes something like this.

There are several references to program IDs (in `lib.rs` and `Anchor.toml`). They depend on keys generated, in our case, for `endpoint` and `oft`. These keys live in `target/deploy`.

If we ignore `target/deploy` (or if you delete it for that matter), next time you run e.g. `anchor test`, new keys will be generated. Tests will fail because the program IDs depend on these keys so you need to run `anchor keys sync`.

What I decided to do is to include three junk keys in the example - `junk-id.json`, `target/deploy/endpoint-keypair.json` and `target/deploy/oft-keypair.json`. This way if you check out devtools and run test, tests will pass.

The other option was way more annoying - generate new keys in `prepare` script and basically having a diff everytime you're making a PR.

Side note: I will also add these junk keys to the list of files to be deleted when you use `create-lz-oapp` so that users are not confused.